tests/serverspec/default_spec.rb

```diff
@@ -3,47 +3,78 @@
 
 package = "sensu-api"
 service = "sensu-api"
-config  = "/etc/sensu-api/sensu-api.conf"
-user    = "sensu-api"
-group   = "sensu-api"
-ports   = [PORTS]
-log_dir = "/var/log/sensu-api"
-db_dir  = "/var/lib/sensu-api"
+config_dir = "/etc/sensu"
+user    = "sensu"
+group   = "sensu"
+ports = [4242]
```

Comment:
it seems it does not have to be an array unless other ports are planned to be added.